features/OrderPlacement.feature

```diff
@@ -0,0 +1,60 @@
+Feature:Order Placement
+
+
+Background:Order Placement
+    Given the user in Homepage and select on the Product
+    When The user added a product to the cart to proceed to a checkout page
+    Then The user is on Checkout landing Page
+
+    Scenario:Validate Empty field in Guest user (Negative scenario)
+    When The user clicks on Next button
+    Then The user should see 8 error messages "This is a required field"
+
+    Scenario: Emailid validation for Guest user (Negative scenario)
+        When  I fill the email field form with "<emailid>"
+        And click Next button
+        Then "Please enter a valid email address" should be displayed
+        Examples:
+            | emailid | 
+            | karthikeyan3071997gmail.com | 
+            | karthikeyan3071997@gmail|
+            | karthikeyan3071997.com|
+
+    Scenario:Validate Empty field in Logged In user (Negative scenario)  
```

Comment:
@karthikeyan7010 How do you make sure user is logged in at this step? if we run this scenario alone with background I don't see anywhere where this user is logged in?